client/apps/settings/client/curations/display/index.jsx

```diff
@@ -1,7 +1,7 @@
 import { set } from 'lodash'
 import PropTypes from 'prop-types'
 import React from 'react'
-import DropDownItem from 'client/components/drop_down/index.jsx'
+import { DropDownList } from 'client/components/drop_down/drop_down_list.jsx'
```

Comment:
Note that you no longer need to append extensions: https://github.com/artsy/positron/blob/master/webpack.config.js#L111